pandas/core/arrays/datetimes.py

```diff
@@ -328,7 +328,6 @@ class DatetimeArray(dtl.DatetimeLikeArrayMixin, dtl.TimelikeOps, dtl.DatelikeOps
     # -----------------------------------------------------------------
     # Constructors
 
-    _attributes = ["freq", "tz"]
```

Comment:
Why are you removing those attributes? (I suppose they are thus not used) It's a left-over from when it shared code with the Index?

Reply:
exactly, leftover from the inheritance days